test/api/gen-extra-close.c

```diff
@@ -15,5 +15,5 @@ int main(void) {
   CHK(yajl_gen_map_close(yg));
   s = yajl_gen_map_close(yg);
 
-  return (yajl_gen_generation_complete == s);
+  return !(yajl_gen_generation_complete == s);
```

Comment:
What's this for?

Reply:
Trying to remember; I'm pretty sure this was related to the missing check of the exit code - this test could effectively never fail.